pkg/test/resourcefixture/testdata/basic/resourcemanager/v1beta1/project/projectmovedfoldertofolder/_generated_object_projectmovedfoldertofolder.golden.yaml

```diff
@@ -24,5 +24,5 @@ status:
     reason: UpToDate
     status: "True"
     type: Ready
-  number: "2589592254"
```

Comment:
Doubt if this is the same as targetId, if so we will probably need to replace that as well (currently it's overridden in the normalize).